### PR TITLE
Feature/tank level

### PR DESCRIPTION
**Pull request by Hunter27**
Added feature functionality for tank data and 24 hour tank graphs

### CODE REVIEW

File: WaterLog_Backend/Procedures.cs [hunk 1]
```diff
@@ -1204,8 +1204,7 @@ public List<MonitorHeat> getMonitorsFaultLevels()
         public async Task<DataPoints<DateTime, double>> getTankGraph(int tankId)
         {
             var dailyTank = await _db
-                            .TankReadings
-                            .Where(a => a.TimeStamp.Day == DateTime.Now.Day && a.TimeStamp.Month == DateTime.Now.Month && a.TimeStamp.Year == DateTime.Now.Year && a.TankMonitorsId==tankId)
+                            .TankReadings.Where(a => a.TimeStamp.Day == DateTime.Now.Day && a.TimeStamp.Month == DateTime.Now.Month && a.TimeStamp.Year == DateTime.Now.Year && a.TankMonitorsId==tankId)
```
Comment (FredJEhlers) [inline WaterLog_Backend/Procedures.cs]: update this line to be something like this 
```
var dailyTank = await _db
	.TankReadings.Where(a => 
		((a.TimeStamp.Day == DateTime.Now.Day) && (a.TimeStamp.Month == DateTime.Now.Month)) &&
		((a.TimeStamp.Year == DateTime.Now.Year) && (a.TankMonitorsId==tankId)))
	.GroupBy(b => b.TimeStamp.Day)
	.ToListAsync();
```

File: WaterLog_Backend/Procedures.cs [hunk 2]
```diff
@@ -1233,8 +1237,7 @@ public List<MonitorHeat> getMonitorsFaultLevels()
         public async Task<List<TankObject>> getObjects()
         {
             var allGrouped  = await _db
-                            .TankReadings
-                            .Where(a => a.TimeStamp.Month == DateTime.Now.Month && a.TimeStamp.Year == DateTime.Now.Year )
+                            .TankReadings.Where(a => a.TimeStamp.Month == DateTime.Now.Month && a.TimeStamp.Year == DateTime.Now.Year )
```
Comment (FredJEhlers) [inline WaterLog_Backend/Procedures.cs]: Update this one as well

File: WaterLog_Backend/Procedures.cs (hunk 1 above)
Comment (FredJEhlers) [reply WaterLog_Backend/Procedures.cs]: Please take careful note of the use of `( )` and how I grouped them logically. 
Also make sure it still works as expected

**Review by FredJEhlers — CHANGES_REQUESTED** [review]

File: WaterLog_Backend/Procedures.cs (hunk 1 above)
Comment (Hunter27) [reply WaterLog_Backend/Procedures.cs]: done

File: WaterLog_Backend/Procedures.cs (hunk 2 above)
Comment (Hunter27) [reply WaterLog_Backend/Procedures.cs]: done

**Review by FredJEhlers — APPROVED** [review]

**Pull request merged**